app/Components/PlayButton/interfaces/play-button-props.interface.ts

```diff
@@ -0,0 +1,8 @@
+import { ReactNode } from 'react';
+
+export interface PlayButtonPropsInterface {
+  icon: string;
+  onClick: () => void;
```

Comment:
I think this is code duplicating use same interface in button mobile and button desktop